lumicks/pylake/file.py

```diff
@@ -187,7 +187,8 @@ def _get_photon_time_tags(self, name):
     def _get_object_dictionary(self, field, cls):
         if field not in self.h5:
             return dict()
-        return {name: cls.from_dataset(dset, self) for name, dset in self.h5[field].items()}
+        object_list = [(name, cls.from_dataset(dset, self)) for name, dset in self.h5[field].items()]
+        return {name: obj for name, obj in object_list if obj is not None}
```

Comment:
AFAIK `cls.from_dataset()` can't ever return `None`. Or at least it shouldn't. It should only signal errors by raising an exception, not returning `None`. I think that this was a mistake in the original `.scans`. If so, you could just omit this part.

Reply:
This is the result of #62 that this PR is based on. Because a `Scan` might have missing metadata it can't be loaded and `from_dataset()` returns None. I was debating on whether that check should also be done for `PointScan` for consistency with `Scan` and `Kymo`. Either way, having all three call `_get_object_dictionary()` is best to avoid duplicated code

Reply:
Ah, I missed that part. Then things are fine here. Just maybe take a look at the changelog line.